framework/src/controller/helpers/validator/index.js

```diff
@@ -0,0 +1,83 @@
+const Ajv = require('ajv');
```

Comment:
No unit tests coverage for this file.

Reply:
Need to add header for this file, and let's keep in mind that in the near future we want to use lisk-validator